docs/api/navigators/StackNavigator.md

```diff
@@ -111,6 +111,7 @@ Title string used by the back button on iOS, or `null` to disable label. Default
 #### `headerBackImage`
 
 The image source (either a remote URL `{{uri: 'path/to/image'}}` or a local file resource  `{require('path/to/image')}`) used by the back button.
+Defaults to './views/assets/back-icon.png'
```

Comment:
Nit: Change single quotes to `

It might be a good idea to include a link to the folder itself in the repo that contains the back-icon.png files.

I would also add that this back icon image is an iOS back button on iOS devices and an Android back button arrow on Android devices.